packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -183,7 +185,11 @@
   flex-shrink: 0;
 }
 
-.dismissalOneline {
+.dismissOneline {
```

Comment:
Chatted offline--things look good now, though it's not quite clear why. This class was used in the .tsx while not being defined in the .scss, but didn't seem to throw any errors at build or runtime; the class just wasn't applied.